daml-script/daml/Daml/Script.daml

```diff
@@ -138,8 +138,8 @@ data ContractMetadata = ContractMetadata
     keyHash : Text -- ^ base 16 encoded byte string
     driverMetadata : Text -- ^ base 16 encoded byte string
 
-toDisclosedContract : Template t => ContractMetadata -> ContractId t -> t -> DisclosedContract 
-toDisclosedContract meta coid arg =  DisclosedContract with
+toDisclosedContract : Template t => ContractId t -> t -> ContractMetadata -> DisclosedContract 
```

Comment:
I forgot to ask in the previous PR; what happens if `arg` doesn't match the contract on the ledger pointed to by `coid`? (assuming this is used in turn as an argument to `submitDisclosing` or similar)

Reply:
If the argument doesn't typecheck, then the participant will reject the command.
If the argument doesn't match the active contract, then the ledger will reject the transaction.